CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+
+## How to contribute
+
+The review of contributions will take place on GitHub [molgenis-emx2 repository](https://github.com/molgenis/molgenis-emx2). A GitHub account is required to contribute. Depending on the type of contributions (issues, features, code), there are different methods for contributing to the MOLGENIS EMX2 software. These are described in the following sections.
+
+### Reporting issues
+
+If you have noticed any issues or something isn't working as expected, first check the following resources.
+
+1. It may be the case that something is missing or needs additional configuration. Please check the [EMX2 documentation](https://molgenis.github.io/molgenis-emx2/#/) for further information.
+2. Check the version of EMX2 you have installed. You may need to update or reinstall EMX2
+3. If that does not resolve the issue, visit the [issues](https://github.com/molgenis/molgenis-emx2/issues) page. Have a look through the issues to see if the issue hasn't been reported or if it was already solved.
+
+If the issue remains unsolved, then open a new issue. We have a [Bug report temport](https://github.com/molgenis/molgenis-emx2/issues/new/choose) that can be used to report issues. Provide as much information as you can including screenshots, error messages, and instructions on how to reproduce the issue. It is also important to provide information about the browser you are using and the MOLGENIS version number that you have installed.
+
+### Feature requests
+
+We welcome any ideas and suggestions for improving EMX2. First, have a look at the [open issues](https://github.com/molgenis/molgenis-emx2/issues) and [open pull requests](https://github.com/molgenis/molgenis-emx2/pulls) to see if there isn't a feature request or something similar. Otherwise, you can submit a request using the [feature request template](https://github.com/molgenis/molgenis-emx2/issues/new/choose). Provide as much detail as you can including links to examples or other resources.
+
+### Code contributions
+
+Anyone is welcome to contribute to the EMX2 code base. We've outlined this process below.
+
+#### Pre-development
+
+We have created several [developer guides](https://molgenis.github.io/molgenis-emx2/#/dev) that provide information on how EMX2 is structured, the coding guidelines, and other practical information. These will help you get started with writing code and to ensure it is consistent with the rest of the code base.
+
+The easiest way to get started is to start with the latest version of EMX2. Clone the main repository and create a new branch.
+
+```bash
+git clone https://github.com/molgenis/molgenis-emx2
+
+# create a new branch
+git switch -c feat/<name-of-feature>
+```
+
+Regarding branch names, we typically use these patterns for naming branches: `feat/*` for a new feature or improvement, and `fix/` for bugs. Keep the branch names short and concise.
```

Comment:
Do we need to mention `chore/*` ?

Reply:
Yes, this should also be in here. Instead, I put a link to to the Basics Guide as it's defined there